src/assets/ikoner/arena.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="30px" height="25px" viewBox="0 0 24 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Ser ut som det har gått fint nå, men kan være lurt om vi får en rutine på å kjøre svg-er gjennom svgo for å prefikse id-er. Se her: https://nav-it.slack.com/archives/CD4MES6BB/p1567087063011200